track_info.c

```diff
@@ -46,8 +46,12 @@ static void track_info_free(struct track_info *ti)
 
 struct track_info *track_info_new(const char *filename)
 {
+	static uint64_t cur_uid = 0;
+	cur_uid++;
```

Comment:
dbus paths are restricted to certain ascii characters. So we would have to encode the paths and then ensure that they are still unique. It's possible but not trivial.